.travis.yml

```diff
@@ -20,5 +20,6 @@ before_install:
   - travis_retry composer self-update
 
 install:
+  - if [[ "$TRAVIS_PHP_VERSION" != "5.3" ]] && [[ "$TRAVIS_PHP_VERSION" != "5.4" ]]; then composer require symfony/psr-http-message-bridge:dev-wip zendframework/zend-diactoros; fi;
```

Comment:
It's not a dev dependency if you use the PSR-7 converter / bridge, and in that context we use the bridge.